fixtures/gen.json

```diff
@@ -1,34 +0,0 @@
-{
```

Comment:
Noting the most recent version of these live in `fixtures/live/` and `fixtures/test/`.